src/monster.cpp

```diff
@@ -82,6 +82,40 @@ void Monster::removeList()
 	g_game.removeMonster(this);
 }
 
+const std::string& Monster::getName() const
+{
+	if (name.empty()) {
+		return mType->name;
+	}
+	return name;
+}
+
+void Monster::setName(std::string name)
```

Comment:
Const reference for name argument?
`void Monster::setName(const std::string& name)`

Reply:
I've inlined it, for whatever reason, with how it is in player class:

https://github.com/otland/forgottenserver/blob/802ea5e92a30f23bf09175319b3d86c00bbe3160/src/player.h#L142-L147

🤷 

Can of course change that to const ref, it that's what wanted.

Reply:
I think const ref is better than copying to move it

Reply:
> I think const ref is better than copying to move it

Technically we can do both